src/modules/addLike.js

```diff
@@ -0,0 +1,14 @@
+import { Movies } from './resource.js';
+import displayLikes from './viewLikes.js';
+
+function addLike(item) {
+  const obj = new Movies();
+  const payload = { item_id: item };
+  obj.addLike(payload).then((response) => {
+    if (response) {
+      displayLikes();
+    }
+  });
+}
```

Comment:
- [ ] Since you are using ES6, It would be better to use arrow functions instead of the word `function`.